### PR TITLE
DTSPO-5506 goldenpath: Deploy Java application

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
This should be in the new goldenpath repo you created?

it should be deployed to backstage not hmcts.github.io.

https://backstage.io/docs/features/techdocs/techdocs-overview

File: source/ways-of-working/deploying-an-app/java/java.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- https://hmcts.github.io/onboarding
+- https://hmcts.github.io/ways-of-working/
+
+This guide uses a pre-configured application from an template which can be deployed from [Backstage](https://backstage.platform.hmcts.net/). Application is deployed to AKS(kubernetes) cluster inside `labs` namespace in sbox/sandbox environment.
+
+## Prerequisites
+
+This guide assumes the following are installed on your local system.
+
+- Docker
```
Comment (timja) [inline source/ways-of-working/deploying-an-app/java/java.html.md.erb:24]: why

File: source/ways-of-working/deploying-an-app/java/java.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      }
+
+   SDS: [config file](https://github.com/hmcts/sharedservices-azure-platform/blob/master/environments/sbox/sbox.tfvars)
+
+
+       {
+        product          = "labs"
+        name             = "#githubreponame without "labs" prefix"
+        custom_domain    = "your-app.sandbox.platform.hmcts.net"
+        backend_domain   = ["firewall-sbox-int-palo-sdsapimgmt.uksouth.cloudapp.azure.com"]
```
Comment (timja) [inline source/ways-of-working/deploying-an-app/java/java.html.md.erb:98]: api mgmt?

File: source/ways-of-working/deploying-an-app/java/java.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        certificate_name = "wildcard-sandbox-platform-hmcts-net"
+      }
+
+
+9) We practise [Gitops](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
```
Comment (timja) [inline source/ways-of-working/deploying-an-app/java/java.html.md.erb:103]:
```suggestion
9) We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
```

File: source/ways-of-working/deploying-an-app/java/java.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+   SDS: values.sbox.template.yaml
+
+
+	java:
+	  # Don't modify below here
+	  image: ${IMAGE_NAME}
+	  ingressHost: ${SERVICE_FQDN}
+
+
+4) Get the PR reviewed and merged.
```
Comment (timja) [inline source/ways-of-working/deploying-an-app/java/java.html.md.erb:59]: no need to get the PR reviewed just merge

avoid acronyms as well if they aren't needed

Comment (mokainos) [reply source/ways-of-working/deploying-an-app/java/java.html.md.erb:59]: I don't think developers has permission to merge the changes by default

**Pull request closed without merging**